addons/test_discuss_full/tests/test_performance.py

```diff
@@ -16,6 +16,11 @@
 class TestDiscussFullPerformance(HttpCase, MailCommon):
     # Queries for _query_count_init_store (in order):
     #   1: internalUserGroupId: ref exists
+    #   4: subtypes:
+    #       - ref exists mt_note
+    #       - ref exists mt_comment
```

Comment:
We don't really need to check existence, `_xmlid_to_res_id` might be sufficient (db is fucked if these don't exist).